cfn.yaml

```diff
@@ -131,4 +131,36 @@ Resources:
       MemorySize: 1536
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java8
-      Timeout: 60
\ No newline at end of file
+      Timeout: 60
+
+  MonitoringSchedulePermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName: !GetAtt Lambda.Arn
+      Principal: events.amazonaws.com
+      SourceArn: !GetAtt MonitoringSchedule.Arn
+
+  MonitoringSchedule:
+    Type: AWS::Events::Rule
+    Properties:
+      Name: !Sub data-lake-alerts-monitoring-schedule-${Stage}
+      ScheduleExpression: cron(0 6 ? * MON-FRI *)
+      State: ENABLED
+      Targets:
+        - Arn: !GetAtt Lambda.Arn
+          Id: ios-friction-screen-scheduler
+          Input: |
+            {
+              "featureId": "friction_screen",
+              "platformId": "ios"
+            }
+        - Arn: !GetAtt Lambda.Arn
+          Id: android-friction-screen-scheduler
+          Input: |
+            {
+              "featureId": "friction_screen",
+              "platformId": "android"
+            }
+    DependsOn:
```

Comment:
You don't need `DependsOn` because Cloudwatch will infer the dependency when it sees the `!GetAtt Lambda.Arn` line above.

Reply:
Thanks, I didn't know about this 👍